templates/user/settings/security/accountlinks.tmpl

```diff
@@ -1,5 +1,25 @@
 <h4 class="ui top attached header">
 	{{.i18n.Tr "settings.manage_account_links"}}
+	{{if .OrderedOAuth2Names}}
+		<div class="ui right">
+			<div class="ui floating1 jump dropdown">
+				<div class="ui blue tiny button">Link Account</div>
+				<div class="menu">
+					{{range $key := .OrderedOAuth2Names}}
+						{{$provider := index $.OAuth2Providers $key}}
+						<a class="item" href="{{AppSubUrl}}/user/oauth2/{{$key}}">
+							<img
+								alt="{{$provider.DisplayName}}{{if eq $provider.Name "openidConnect"}} ({{$key}}){{end}}"
+								title="{{$provider.DisplayName}}{{if eq $provider.Name "openidConnect"}} ({{$key}}){{end}}"
+								class="{{$provider.Name}} oauth-login-image"
```

Comment:
```suggestion
								alt="{{$provider.DisplayName}}"
```

Only `alt` is required by best practice (every img should have the alt message)

`title` doesn't help here because user couldn't get more information when hovering

`class` seems incorrect here, it seems to be copied from somewhere else, it's only used by JS to show/hide some HTML elements (see `initUserAuthOauth2` in JS).